### DISCUSSION TITLE
Can I port openTcs to the springBoot framework?

### DISCUSSION THREAD

**Discussion by CSXYA** [openTCS/opentcs#88] [Q&A]
Can I port openTcs to the springBoot framework?

**Comment by Superfortunate** [comment]
I'd like to know too.

**Comment by Superfortunate** [comment]
Do you have a solution, paid

**Comment by swltr** [comment]
@CSXYA and @Superfortunate openTCS itself does not use Spring Boot. What do you mean by _porting_ openTCS to Spring Boot? Of course you can fork openTCS and modify the source code to integrate Spring Boot. But I assume you mean something else.
